doc/Custom-types.md

```diff
@@ -171,6 +171,17 @@ myApp.config(['FieldViewConfigurationProvider', function(fvp) {
 }]);
 ```
 
+To use this field view, alter the field class and set its `_type` property:
+```js
+class AmountField extends NumberField {
+    constructor(name) {
+        super(name);
+        this._type = 'amount';
```

Comment:
Done.

I thought the intention of the documentation was to show that the field view definition was optional, that's why I didn't add it to line 90 in the first place.